template-manager-blueprint/src/main/java/com/sequenceiq/cloudbreak/clusterdefinition/filesystem/FileSystemConfigQueryService.java

```diff
@@ -84,12 +86,10 @@ public void init() {
             try {
                 configQueryEntry.setProtocol(fileSystemType.getProtocol());
                 configQueryEntry.setDefaultPath(generateConfigWithParameters(configQueryEntry.getDefaultPath(), fileSystemType, templateObject));
-            } catch (IOException e) {
-                configQueryEntry.setDefaultPath(configQueryEntry.getDefaultPath());
+            } catch (IOException ignored) {
```

Comment:
where do you set the default path? it is set in the catch clause

Reply:
`configQueryEntry.setDefaultPath(configQueryEntry.getDefaultPath());` uses the same object to get and set.